packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.constant.js

```diff
@@ -7,21 +7,22 @@ export const UNAVAILABLE_PCC_CODE = [400, 404];
 
 export const VCD_PLAN_CODE = 'pcc-option-to-vcd-migration';
 
-export const MANAGED_PCC_MIGRATION_STATUS = {
+export const VCD_MIGRATION_STATUS = {
   ENABLING: 'enabling',
+  ENABLED: 'enabled',
   UNKNOWN: 'unknown',
 };
 
-export const MANAGED_VCD_MIGRATION_STATUS = {
+export const PCC_MIGRATION_STATUS = {
   TO_MIGRATE: 'TO_MIGRATE',
   MIGRATED: 'MIGRATED',
   NOT_ALLOWED: 'NOT_ALLOWED',
 };
 
-export const MANAGED_VCD_MIGRATION_STATE = {
-  0: MANAGED_VCD_MIGRATION_STATUS.NOT_ALLOWED,
-  1: MANAGED_VCD_MIGRATION_STATUS.TO_MIGRATE,
-  2: MANAGED_VCD_MIGRATION_STATUS.MIGRATED,
+export const PCC_MIGRATION_STATE = {
+  0: PCC_MIGRATION_STATUS.NOT_ALLOWED,
+  1: PCC_MIGRATION_STATUS.TO_MIGRATE,
+  2: PCC_MIGRATION_STATUS.MIGRATED,
 };
 
 export const VCD_GUIDE_LINKS = {
```

Comment:
We not separe PCC and VCD Guide links ?

Reply:
All the guides are use only for PCC  here